core/src/utils.jl

```diff
@@ -208,19 +279,21 @@ end
 
 "Return the bottom elevation of the basin with index i"
 function basin_bottom_index(basin::Basin, i::Int)::Float64
-    # get level(storage) interpolation function
-    itp = basin.level[i]
-    # and return the first level in the underlying table, which represents the bottom
-    return first(itp.u)
+    # TODO: Is this function still needed now that it is so simple?
+    # get discrete level values
+    level_discrete = basin.level[i]
+    # and return the first level in this vector, representing the bottom
+    return first(level_discrete)
 end
 
 "Return the bottom elevation of the basin with index i"
 function basin_bottom(basin::Basin, node_id::Int)::Float64
+    # TODO: Is this function still needed now that it is so simple?
```

Comment:
It's still about 8 lines of code, making it easier to read: `bottom_a = basin_bottom(basin, basin_a_id)`